authors/apps/authentication/tests/test_login_register.py

```diff
@@ -19,15 +19,15 @@ def setUp(self):
             self.user_data,
             format="json")
 
-    def test_api_can_register_a_user(self):
```

Comment:
There was no need of deleting and rewriting the same test. The deleted test was reusing data in the base test file instead of rewriting similar data in the test.

Reply:
i have solved this @collinewait